### ISSUE TITLE
Writeroom does not play well with git-gutter in TTY

### ISSUE THREAD

**Issue by Shooooooooo** [hlissner/doom-emacs#2859]
This is probably not a game breaking bug for me. But, it's nice if there is a quick fix.

**What did you expect to happen?**
Like the GUI frame, buffer should be centered once `writeroom-mode` is enabled.

**What actually happened?**
If `git-gutter-mode` is enabled at the same time, turning on `writeroom-mode` will make the buffer centered for a split second, and then aligned to the left border again.

**Additional details:**
I believe this is a similar issue to joostkremers/writeroom-mode#40, but for TTY.
TLDR I guess is some conflict about both modes trying to compute margins.
Maybe the workaround is, for TTY, have a hook which disables git-gutter when turning on writeroom.

**Steps to reproduce:**
Have git-gutter and writeroom-mode both enabled at the same time in TTY with a locally modified git tracked file.

**Comment by Shooooooooo** [comment]
Closing this issue. The issue appeared to be gone for emacs version greater than 27. I suspect it's an emacs TTY margin issue.
